src/components/timeline/Markers.js

```diff
@@ -198,16 +374,6 @@ class TimelineMarkersImplementation extends React.PureComponent<Props, State> {
     });
   };
 
-  componentDidUpdate(prevProps: Props, prevState: State) {
-    if (
-      prevProps !== this.props ||
-      prevState.hoveredItem !== this.state.hoveredItem ||
-      prevState.mouseDownItem !== this.state.mouseDownItem
-    ) {
-      this._scheduleDraw();
-    }
-  }
```

Comment:
This is the `componentDidUpdate` with the guarding `if` condition. I believe the `props` object changes all the time once we use dispatch props, because the function is wrapped with `dispatch`. But I'm not sure and I couldn't find the code responsible for this in react-redux.